src/platform/forms/save-in-progress/SaveInProgressErrorPage.jsx

```diff
@@ -19,6 +19,15 @@ import { toggleLoginModal } from '../../site-wide/user-nav/actions';
 // For now, this only handles loading errors, but it could feasibly be reworked
 //  to handle save errors as well if we need it to.
 class SaveInProgressErrorPage extends React.Component {
+  componentDidMount() {
+    if (
+      this.props.loadedStatus === LOAD_STATUSES.notAttempted &&
+      document.referrer.includes('/auth/login/callback/')
+    ) {
+      this.props.router.push(this.props.location.basename);
```

Comment:
I believe there's also a `replace` method that we could use instead of `push`. That will replace the `/error` entry in the browser history with the page we're sending the user to, which will make more sense to them if they hit the back button, I think. Otherwise, if they click back, we'll run this logic again and send them to the intro page again.